### PR TITLE
Rubocop updates

### PR DESCRIPTION
**Pull request by kaylareopelle**
Innovation meeting project: 
* Audit of some linters turned off in the initial run and addressing them. These changes are all from safe autocorrect.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -661,9 +606,9 @@ Lint/EmptyConditionalBody:
   Enabled: false
 
 # Disabling for now
-Lint/EmptyEnsure:
-  Enabled: false
-  # AutoCorrect: true
+# Lint/EmptyEnsure:
```
Comment (hannahramadan) [inline .rubocop.yml:609]: Do we want to remove this entirely vs comment out?

File: test/performance/lib/performance/result.rb [hunk 2]
```diff
@@ -6,8 +6,8 @@
 
 module Performance
   class Result
-    attr_reader :test_name, :measurements, :tags, :timer, :artifacts
-    attr_accessor :exception, :iterations
+    attr_reader :test_name, :measurements, :tags, :timer, :artifacts, :exception
```
Comment (fallwith) [inline test/performance/lib/performance/result.rb:9]: I'm surprised there isn't a "these should be alphabetized" cop in play.

**Review by fallwith — APPROVED** [review]

File: test/performance/lib/performance/result.rb (hunk 2 above)
Comment (kaylareopelle) [reply test/performance/lib/performance/result.rb:9]: It might be on the list of disabled cops that we didn't get to yesterday!

**Review by tannalynn — APPROVED** [review]

File: .rubocop.yml (hunk 1 above)
Comment (kaylareopelle) [reply .rubocop.yml]: Great catch! Fixed in: 03134c1433aa21e2ffef9cc67cc47ef1ce3f92f4

**Review by fallwith — APPROVED** [review]

**Pull request merged**